Classes/Item.lua

```diff
@@ -567,10 +567,20 @@ function ItemClass:GetModSpawnWeight(mod, extraTags)
 			return false
 		end
 
+		local function HasMavenInfluence(modAffix)
+			if modAffix:match("Elevated") then
+				return true
+			end
+			return false
```

Comment:
The if statement is redundant here

```suggestion
			return modAffix:match("Elevated")
```